node_modules/col-assets/lib/api.js

```diff
@@ -55,7 +55,8 @@ var UserConstants = require('col-users/lib/constants');
 var getAssetProfile = module.exports.getAssetProfile = function(ctx, id, incrementViews, callback) {
   var opts = {
     'incrementViews': incrementViews,
-    'includeWhiteboardElements': true
+    'includeWhiteboardElements': true,
+    'includeWhiteboardUsages': true
   };
```

Comment:
I almost went that way, @nicolaasmatthijs, but thought that the front end might want to associate assets with whiteboards for other reasons down the line. We can move it all to the back end if you'd prefer, though.